static/src/stylesheets/module/content/_article-minute.scss

```diff
@@ -571,8 +571,12 @@ $minute-layout-margin-nudge: gs-span(1);
                     width: 100%;
                 }
 
-                .gu-image {
+                .element--inline__outer {
```

Comment:
It's normally `block__element--modifier`.

To stop creating layout decisions (i.e. `element--inline__outer` should always be in `element--inline`) adding complexity, I would say just create the style and name it, aka, `inline-outer`, which doesn't specify usage.

That's my personal understanding of BEM though, some people like to use it more complexly.

@sndrs opinion?